servo-media-derive/Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
-name = "servo_media_derive"
+name = "servo-media-derive"
 version = "0.1.0"
 authors = ["Fernando Jiménez Moreno <ferjmoreno@gmail.com>"]
 
 [lib]
-name = "servo_media_derive"
```

Comment:
This needs to be reverted.

Reply:
Oops. You're right. This is reverted now